stack/loader.go

```diff
@@ -103,25 +103,24 @@ func (l Loader) Set(dir string, s S) {
 	l.stacks[dir] = s
 }
 
-// LoadAll loads all the stacks in the dirs directories. If dirs are relative
-// paths, then basedir is used as base.
-func (l Loader) LoadAll(root string, basedir string, dirs ...string) ([]S, error) {
+// LoadAll loads all the stacks in the dirs project paths. If dirs are relative,
+// then wd is used as working directory to compute their absolute paths.
+func (l Loader) LoadAll(root string, wd string, dirs ...string) ([]S, error) {
```

Comment:
```suggestion
// LoadAll loads all the stacks in the dirs paths. For each dir in dirs: 
// - If it is relative, it will be considered relative to wd, path = wd + dir
// - If it is absolute, it will be considered absolute in relation to the given root, path = root + dir
func (l Loader) LoadAll(root string, wd string, dirs ...string) ([]S, error) {
```

Reply:
it did improve, thanks man. But was still confused about the role of root, so tried to make that clear also.